rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+
+The following rules apply to a setRequest containing `origin` CLI.  TODO:
+Special considerations for mixing native schema are currently undefined.
+
+* If a `SetRequest` contains `origin` `cli` and `origin` `openconfig`, the CLI
+  MUST be given precedence over the OpenConfig origin.  This precedence is
+  independent of the order in which origin CLI appears.  This overrides the
```

Comment:
Instead of "This precedence is independent of the order in which origin CLI appears",
how about saying "It's invalid for a SetRequest to contain oc/cli origins that do not fall into the following categories...", which I think has the benefit
1. forces the user to explicitly think of their expectation.
2. makes it easier to debug (an grpc error message is better than verifying device behavior after an 'accepted' SetRequest).

Reply:
gNMI specifically requires order based precedence.  This special case for origin CLI overrides this by asserting CLI always takes precedence over OC, regardless of order.